content/en/tracing/service_catalog/guides/_index.md

```diff
@@ -9,4 +9,6 @@ private: true
     {{< nextlink href="tracing/service_catalog/guides/understanding-service-configuration" >}}Understanding Your Service Configuration{{< /nextlink >}}
     {{< nextlink href="tracing/service_catalog/guides/validating-service-definition" >}}Validating Service Definition YAMLs{{< /nextlink >}}
     {{< nextlink href="/tracing/service_catalog/guides/upstream-downstream-dependencies" >}}See Upstream and Downstream Dependencies During an Active Incident{{< /nextlink >}}
+    {{< nextlink href="/tracing/service_catalog/guides/integrating-with-ci-visibility" >}}Streamline the Development Lifecycle in Service Catalog with CI Visibility{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="/tracing/service_catalog/guides/streamlining-development-lifecycle-with-ci-visibility/" >}}Streamline the Development Lifecycle in Service Catalog with CI Visibility{{< /nextlink >}}
```

Reply:
`/tracing/service_catalog/guides/integrating-with-ci-visibility` results in a 404, assuming this is meant to link to the new page `/tracing/service_catalog/guides/streamlining-development-lifecycle-with-ci-visibility/` ?

Reply:
It does, my bad (changed the name of the file in a last round of reviews)